bindings/pyroot/pythonizations/python/ROOT/__init__.py

```diff
@@ -184,15 +184,4 @@ def cleanup():
         facade.__dict__["app"].keep_polling = False
         facade.__dict__["app"].process_root_events.join()
 
-    if "libROOTPythonizations" in sys.modules:
```

Comment:
Why do we even do this `if "libROOTPythonizations" in sys.modules` check now that we don't use `gROOT` from `libROOTPythonizations` anymore? Maybe always running the code branch helps...

Reply:
Yeah that should not be needed anymore, but probably it's orthogonal. I will check if it makes a difference anyway